.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
-- repo: https://gitlab.com/pycqa/flake8
+- repo: https://github.com/PyCQA/flake8
```

Comment:
I made the suggested change to the link from [here](https://github.com/AICoE/aicoe-ci/issues/53)